api/src/pcapi/connectors/api_allocine.py

```diff
@@ -7,9 +11,27 @@ class AllocineException(Exception):
     pass
 
 
+logger = logging.getLogger(__name__)
+
 ALLOCINE_API_URL = "https://graph-api-proxy.allocine.fr/api/query"
 
 
+def _extract_allocine_id_from_allocine_movie_list(movie_list: dict, path: tuple[int | str, ...]) -> str | None:
+    try:
+        # If the validation error comes from a movie, the path should be
+        # ["movieList", "edges", movie_index, ...]
+        # Otherwise, there is no allocine id to retrieve.
+        obj = movie_list
+        for field in path[:3]:
+            obj = obj[field]
+        allocine_id = obj["node"]["internalId"]
+    except (KeyError, IndexError):
+        logger.error("Error extracting allocine id from movie list")
```

Comment:
En loggant avec `exception`, on aurait le traceback complet dans Sentry, dont les valeurs des variables, qui seront sûrement utiles au debug.

```suggestion
        logger.exception("Error extracting allocine id from movie list")
```